Blockzilla/Modules/WebView/WebViewController.swift

```diff
@@ -81,10 +81,15 @@ class WebViewController: UIViewController, WebController {
             self.delegate?.webController(self, didUpdateEstimatedProgress: webView.estimatedProgress)
         }
 
-        ContentBlockerHelper.getBlockLists { lists in
-            DispatchQueue.main.async {
-                lists.forEach(self.browserView.configuration.userContentController.add)
-            }
+        ContentBlockerHelper.shared.getBlockLists { lists in
+            self.reloadBlockers(lists)
+        }
+    }
+
+    @objc private func reloadBlockers(_ blockLists: [WKContentRuleList]) {
+        DispatchQueue.main.async {
```

Comment:
No issue doing this on main thread?

Reply:
@Sdaswani I was getting runtime warnings from Xcode when I wasn't running it on the main thread.